File: project/angular_project/src/app/models/city.ts
import { Country } from "./country";

export class City {
  city_id: number;
  name: string;
  size: number;
  country: Country;

  constructor() {
    this.city_id = 0;
    this.name = '';
    this.size = 0;
    this.country = new Country(); 
  }
}

File: project/src/main/java/com/example/project/entities/City.java
package com.example.project.entities;

import java.io.Serializable;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.DBRef;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.Data;

@Data
@Document(collection = "City")
public class City implements Serializable{
	@Id
	private int city_id;
	private String name;
	private int size;
    private Country country;
	
}